.travis.yml

```diff
@@ -5,8 +5,14 @@ notifications:
     on_success: never
     on_failure: change
 
+env:
+  global:
+    - CXX=g++-4.8
+
 node_js:
-  - 0.10
+  - stable
+  - v4.2
+  - v0.10
```

Comment:
I'm okay removing `v0.10` since we are completely moving past support for it, any concerns there?